.factory/automation.yml

```diff
@@ -70,196 +75,54 @@ build:
-#        bazel test //test/behaviour/reasoner/tests/relation_inference:test --test_output=streamed
-#        bazel test //test/behaviour/reasoner/tests/rule_interaction:test --test_output=streamed
-#        bazel test //test/behaviour/reasoner/tests/schema_queries:test --test_output=streamed
-#        bazel test //test/behaviour/reasoner/tests/type_hierarchy:test --test_output=streamed
-#        bazel test //test/behaviour/reasoner/tests/value_predicate:test --test_output=streamed
-#        bazel test //test/behaviour/reasoner/tests/variable_roles:test --test_output=streamed
-#    test-benchmark-reasoner:
+#        bazel test //test/behaviour/query/language:test_define --test_output=streamed
+#        bazel test //test/behaviour/query/language:test_undefine --test_output=streamed
+#        bazel test //test/behaviour/query/language:test_redefine --test_output=streamed
+#    test-behaviour-functions:
 #      image: vaticle-ubuntu-22.04
-#      filter:
-#        owner: typedb
-#        branch: [master, development]
 #      dependencies: [build]
 #      command: |
 #        bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
-#        bazel test //test/benchmark/reasoner/iam/basic:test-basic --test_output=streamed
-#        bazel test //test/benchmark/reasoner/iam/complex:test-conjunction-structure --test_output=streamed
-#        bazel test //test/benchmark/reasoner/iam/complex:test-complex-rule-graph --test_output=streamed
-#        bazel test //test/benchmark/reasoner/iam/complex:test-language-features --test_output=streamed
-#        bazel test //test/benchmark/reasoner/iam/complex:test-large-data --test_output=streamed
-#        bazel test //test/benchmark/reasoner/iam/complex:test-real-queries --test_output=streamed
-#    test-assembly-linux-targz:
-#      image: vaticle-ubuntu-20.04
-#      filter:
-#        owner: typedb
-#        branch: [master, development]
-#      dependencies: [build, build-dependency, test-unit, test-integration, test-behaviour-connection, test-behaviour-concept, test-behaviour-query-read, test-behaviour-query-write, test-behaviour-query-definable, test-behaviour-reasoner]
-#      command: |
-#        bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
-#        bazel test //test/assembly:assembly --test_output=streamed
-#    test-assembly-docker:
-#      image: vaticle-ubuntu-22.04
-#      filter:
-#        owner: typedb
-#        branch: [master, development]
-#      dependencies: [build, build-dependency, test-unit, test-integration, test-behaviour-connection, test-behaviour-concept, test-behaviour-query-read, test-behaviour-query-write, test-behaviour-query-definable, test-behaviour-reasoner]
-#      command: |
-#        bazel run @vaticle_dependencies//tool/bazelinstall:remote_cache_setup.sh
-#        bazel test //test/assembly:docker --test_output=streamed
-#    deploy-artifact-snapshot:
```

Comment:
I don't have a docker assembly test yet. Should I implement one now?
I do have an assembly test, run in CircleCI before deployment - though the deployments currently run in parallel so we would deploy an incomplete set.

Reply:
Yes i think Docker being prioritised is a good shout